docroot/themes/custom/dcp/sass/pages/_session_listing.scss

```diff
@@ -113,3 +113,8 @@
     background: $color--cat-beyond;
   }
 }
+.session-listing--my .views-field-edit-node {
+  position: absolute;
```

Comment:
@malabya - Is the position: relative implemented respectively to the position: absolute?

Reply:
`.views-row` has a position relative applied to it. So, the `.views-field-edit-node` will be absolute, relative to the `.views-row` wrapper.